Makefile

```diff
@@ -118,6 +118,9 @@ build.version:
 	@mkdir -p $(OUTPUT_DIR)
 	@echo "$(VERSION)" > $(OUTPUT_DIR)/version
 
+# Change how CRDs are generated for CSVs
+build.common: export MAX_DESC_LEN=0 # sets the description length to 0 since CSV cannot be bigger than 1MB
+build.common: export NO_OB_OBC_VOL_GEN=true
```

Comment:
Should exporting these two vars be only next to the `make crds` or csv gen? It also seems odd to define `build.common` three times.

Reply:
since we include the csv in rook image for that we need to pass this export for not generating obc rbac and also removing the description. We'll remove this all soon as csv in images are not required in downstream now.

I kept `build.common` separate for exporting variable and calling other make command